<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pantalla</class>
 <widget class="QMainWindow" name="pantalla">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pantalla</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGenerarHumanos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Humanos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfGenerarHumanos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="grbConsultas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>481</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rbtnConsultaID">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar por ID</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtnConsultaFamilia">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar familia de un ID</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtnConsultaAmigosAmigos">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar amigos de amigos de un ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConsultar">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txfConsultaID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnImprimirArbol">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimir el Arbol</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxDeporte">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Atletismo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Badminton</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Baloncesto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Balonmano</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Boxeo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Canotaje</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ciclismo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ecuestre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Esgrima</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Futbol</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gimnasia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Halterofilia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hockey</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Judo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Natacion</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pentatlon Moderno</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Taekwondo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tenis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tenis de mesa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tiro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tiro con arco</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triatlon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vela</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Voleibol de playa</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnConsultarDeporte">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar Deporte</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConsultarHumanos">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad de Humanos Vivos, Muertos y Salvados</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbVillanos">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>591</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Villanos</string>
    </property>
    <widget class="QPushButton" name="btnThanos">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Thanos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEbonyMaw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ebony Maw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCorvusGlaive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Corvus Glaive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMidnight">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Midnight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBlackDwarf">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Black Dwarf</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNebula">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nebula</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConsultaVillanos">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar Villanos</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbVengadores">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>591</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Vengadores</string>
    </property>
    <widget class="QPushButton" name="btnSpiderman">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Spiderman</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThor">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Thor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAntMan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ant Man</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIronMan">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ironman</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConsultarVengadores">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar Vengadores</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
